projects/TPVFormer/config/tpvformer_8xb1-2x_nus-seg.py

```diff
@@ -0,0 +1,317 @@
+_base_ = ['../../../configs/_base_/default_runtime.py']
```

Comment:
modify the name of directory
projects/TPVFormer/config -> projects/TPVFormer/configs